torch/_dynamo/utils.py

```diff
@@ -262,13 +262,6 @@ def setup_compile_debug():
     compile_debug = bool(os.environ.get("TORCH_COMPILE_DEBUG", False))
 
     if compile_debug:
-        torch._logging.set_logs(
```

Comment:
@mlazos Actually, I'm not sure how to prevent the logs from being written to stdout while still writing them to a file. Can you provide some guidance here?

Reply:
I'm just going to turn off `debug_log` for now.